src/python/pants/backend/jvm/tasks/coverage/cobertura.py

```diff
@@ -62,7 +65,7 @@ def cobertura_jar(**kwargs):
 
     register_jvm_tool(register, 'cobertura-report', classpath=[cobertura_jar()])
 
-  def __init__(self, settings):
+  def __init__(self, settings, targets, execute_java_for_targets):
```

Comment:
Some doc on argument type and meaning (particularly `execute_java_for_targets`) would be most helpful!